src/libs/OptionsListUtils.js

```diff
@@ -25,6 +25,19 @@ let preferredLocale;
 // will update when the OptionsListUtils re-loads. But will stay the same color for the life of the JS session.
 const defaultAvatarForUserToInvite = getDefaultAvatar();
 
+/**
+ * Adds expensify SMS domain (@expensify.sms) if login is a phone number and if it's not included yet
+ *
+ * @param {String} login
+ * @return {String}
+ */
+function addSMSDomainIfPhoneNumber(login) {
+    if (Str.isValidPhone(login) && !Str.isValidEmail(login)) {
+        return login + CONST.SMS.DOMAIN;
+    }
+    return login;
+}
```

Comment:
This _could_ go nicely in expensify-common, but I wanted to keep this in one PR since it's a deploy blocker